README.md

```diff
@@ -4,7 +4,7 @@ Asynchronous HTTP client used for communication with the Discord REST API.
 
 ## Requirements
 
-- PHP >=7.2
+- PHP >=8
```

Comment:
Upped this to match DiscordPHP, 7.x versions are all EOL anyway

Reply:
Since we still supply 7.x series as stable, not yet please, you can bump it to 7.4